pkg/decoder/nomadxs/v1/port1.go

```diff
@@ -53,3 +60,41 @@ type Port1Payload struct {
 	MagnetometerYAxis  float32 `json:"magnetometerYAxis"`
 	MagnetometerZAxis  float32 `json:"magnetometerZAxis"`
 }
+
+var _ common.Position = &Port1Payload{}
+
+func (p Port1Payload) GetLatitude() float64 {
+	return p.Latitude
+}
+
+func (p Port1Payload) GetLongitude() float64 {
+	return p.Longitude
+}
+
+func (p Port1Payload) GetAltitude() *float64 {
+	return &p.Altitude
+}
+
+func (p Port1Payload) GetSource() common.PositionSource {
+	return common.PositionSource_GNSS
+}
+
+func (p Port1Payload) GetCapturedAt() *time.Time {
+	capturedAt, err := time.Parse(time.RFC3339,
+		fmt.Sprintf("%d-%02d-%02dT%02d:%02d:%02dZ",
```

Comment:
Couldn't we avoid string parsing by initialising a zero time, then adding all the separate values to it?